### ISSUE TITLE
Show Name statistics displays User #s instead of links to Users' names

### ISSUE THREAD

**Issue by JoeCohen** [MushroomObserver/mushroom-observer#1795]
<img width="473" alt="Screenshot 2024-01-12 at 9 06 24 AM" src="https://github.com/MushroomObserver/mushroom-observer/assets/45460/d6bd0147-e3cc-4e71-9a75-5e1fbee55e78">

To replicate:
- Show any Name
- Scroll down to statistics area
Result: creator and last editor are displayed as "User #_nnn_".
It should display the Users' names, preferably as links to those users.
See screenshot above. (User # 123 is Alan Rockefeller.)

Same problem in Location statistics.

**Comment by JoeCohen** [comment]
See app/helpers/footer_helper.rb
It renders in a paragraph with classes "small footer-view-stats mt-3". The p does not have an ID.

**Comment by JoeCohen** [comment]
Probably unfair to call this a bug. It's a side effect of this:
```ruby
    # This is yet another db lookup of users - let's try skipping it.
    # latest_user = User.safe_find(versions.latest.user_id)
```
In order to get the latest user's name, we need to know the latest user. And we need to look that up because a Version is not an AR Object. 
I need to think about how to proceed. Maybe the footer shows only
created, updated, name of creator (which is object.name -- doesn't require another lookup)

**Comment by JoeCohen** [comment]
@nimmolo:
TLDR:  I propose showing only Created By with the user's name, created date, last updated.

Longer story: 
Showing user #s is not terribly useful and looks strange.
What would be most useful to me would be restoring the old behavior, particularly the names of the creator and last editor.
But - (1) judging by the lack of user complaints, I'm the only one who cares., and (2) I think that restoring the old behavior would require restoring the old lookup (or maybe a lot of coding). I don't thing it's worth the trouble.
So let's keep it simple. Display `object.created_at`, `object.updated_at`, and `object.user.name.`

**Comment by nimmolo** [comment]
@JoeCohen That sounds good. Let's check if `object.user.name` isn't also a lookup, though. The issue being, the caller of this method (it could be any template) may not necessarily have eager loaded the object.user, and then the footer is stuck looking it up. 

How about this: We could make the requirement more apparent by making `user` (object.user) a param of the method. Then the caller would at least realize that they are looking it up if they don't have it already. 

What was suprising to me in auditing these object footer methods is that they did so many lookups of associations that the calling view or controller could easily eager-load, or may have already eager-loaded.

**Comment by JoeCohen** [comment]
Fixed #1840
